INSTALL.rst

```diff
@@ -0,0 +1,187 @@
+#########################
+Cyclus Installation Guide
```

Comment:
Is there some reason the "Installing Dependencies" link gives the formatted text, whereas the link to the installation instructions here gives the raw text? (I know this isn't the intention but I can't figure out why this happens)